sormas-backend/src/main/resources/sql/sormas_schema.sql

```diff
@@ -4248,5 +4248,11 @@ ALTER TABLE contact ADD COLUMN additionaldetails varchar(512);
 ALTER TABLE contact_history ADD COLUMN additionaldetails varchar(512);
 
 INSERT INTO schema_version (version_number, comment) VALUES (208, '2020-05-11 Add additionalDetails to contact #1933');
-                                                                                                                        
+
+-- 2020-05-07 Add samples to contacts #1753
+ALTER TABLE samples ADD COLUMN associatedcontact_id bigint;
```

Comment:
Add the new column to the history table as well

Reply:
ok